web/client/components/TOC/fragments/SettingsModal.jsx

```diff
@@ -116,16 +115,6 @@ const SettingsModal = React.createClass({
             <Tab eventKey={3} title={<Message msgId="layerProperties.style" />} disabled>Tab 3 content</Tab>
           </Tabs>);
         const footer = (<span role="footer">
-            {this.props.includeCloseButton ? <Button bsSize={this.props.buttonSize} onClick={this.onClose}>{this.props.closeText}</Button> : <span/>}
-            {this.props.includeDeleteButton ?
```

Comment:
I would not remove this piece of code, just set includeDeleteButton to false in the plugin code.